src/Robot.elm

```diff
@@ -0,0 +1,89 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)

```

Comment:
The name of the module should correspond to the name of the file (in this case - `module Robot ...`)